src/service/vsync-impl.js

```diff
@@ -103,11 +125,11 @@ export class Vsync {
    * will be undefined.
    *
    * @param {!VsyncTaskSpecDef} task
-   * @param {!VsyncStateDef=} opt_state
+   * @param {VsyncStateDef=} opt_state
    */
   run(task, opt_state) {
     this.tasks_.push(task);
-    this.states_.push(opt_state || {});
+    this.states_.push(opt_state);
```

Comment:
I found 2 places. Fixed them up.

This API sucks. Should have just let measure return an object.